dspace-api/src/test/data/dspaceFolder/config/spring/api/iiif-processing.xml

```diff
@@ -6,7 +6,6 @@
 
     <bean id="iiifCanvasDimensionServiceFactory" class="org.dspace.iiif.canvasdimension.factory.IIIFCanvasDimensionServiceFactoryImpl"/>
     <bean class="org.dspace.iiif.canvasdimension.IIIFCanvasDimensionServiceImpl" scope="prototype"/>
-    <bean class="org.dspace.iiif.MockIIIFApiQueryServiceImpl" id="org.dspace.iiif.IIIFApiQueryServiceImpl"
+    <bean class="org.dspace.iiif.MockIIIFApiQueryServiceImpl" id="org.dspace.iiif.IIIFApiQueryService:q"
```

Comment:
@mspalti : It's likely I'm missing something in this PR, but why did you change the `id` here to end in ":q"?  This doesn't seem related to the other changes.